mobile/src/board/BoardPages.tsx

```diff
@@ -190,6 +190,7 @@ export function GetById() {
       <BoardView
         remainingTile={
```

Comment:
its a little confufing having one time a prop "remainingTile" that is a component, and elsewhere another variable named "remainingTile" whice is a boolean
Maybe you should change the name of the boolean for something like "hasRemainingTile"

Reply:
Will change in future PR